xbmc/video/windows/GUIWindowVideoBase.h

```diff
@@ -43,6 +43,8 @@ class CGUIWindowVideoBase : public CGUIMediaWindow, public IBackgroundLoaderObse
   virtual ~CGUIWindowVideoBase(void);
   virtual bool OnMessage(CGUIMessage& message) override;
   virtual bool OnAction(const CAction &action) override;
+  virtual void OnInitWindow() override;
+  virtual void OnDeinitWindow(int nextWindowID) override;
```

Comment:
@Paxxi: Is that 100% accurate? If you have another class inherit from this class and want them to be able to override the same method you still need to put the `virtual` there or not? In this case there's `CGUIWindowVideoNav` which derives from `CGUIWindowVideoBase` which derives from `CGUIMediaWindow` and so on. For `CGUIWindowVideoNav` to be able to properly override `OnInitWindow()` et al. you need to define them virtual in `CGUIWindowVideoBase`.